packages/zoe/test/swingsetTests/zoe/vat-dave.js

```diff
@@ -89,8 +89,8 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
         details`source code didn't match at "asset: give.Asset,"`,
       );
       assert(
-        installationCode.includes('firstOfferExpected'),
-        details`source code didn't match at "firstOfferExpected"`,
+        installationCode.includes('firstOfferHandle, offerHandle),'),
```

Comment:
in the new-SES branch, `installationCode` will be replaced by `installationBundle.source`, but the contents of a bundle are opaque.. tests can peek, of course, but let's make sure that downstream developers are encouraged to do a deep-structure comparison between the bundle they submitted and the (whole) bundle returned by `getInstallation`, because the behavior depends upon more than just the one property (if that property is even present, in some future form of the bundle)